policy/manager.go

```diff
@@ -39,9 +40,12 @@ func NewManager(log hclog.Logger, ps map[SourceName]Source, pm *manager.PluginMa
 
 // Run starts the manager and blocks until the context is canceled.
 // Policies that need to be evaluated are sent in the evalCh.
-func (m *Manager) Run(ctx context.Context, evalCh chan<- *Evaluation) {
+func (m *Manager) Run(ctx context.Context, evalCh chan<- *Evaluation, mInt time.Duration) {
```

Comment:
Would `mInt` be better as part of the `Manager` `struct`? Maybe inside some `config` field.

Reply:
Yeah good shout, that makes more sense.